common/djangoapps/third_party_auth/config/waffle.py

```diff
@@ -18,8 +17,21 @@
 # .. toggle_target_removal_date: 2021-04-31
 # .. toggle_warnings: None.
 # .. toggle_tickets: ENT-4034
-ENABLE_MULTIPLE_SSO_ACCOUNTS_ASSOCIATION_TO_SAML_USER = LegacyWaffleSwitch(
-    _WAFFLE_SWITCH_NAMESPACE,
-    'enable_multiple_sso_accounts_association_to_saml_user',
-    __name__
+ENABLE_MULTIPLE_SSO_ACCOUNTS_ASSOCIATION_TO_SAML_USER = WaffleSwitch(
+    f'{WAFFLE_NAMESPACE}.enable_multiple_sso_accounts_association_to_saml_user',
+    module_name=__name__
+)
+
+# .. toggle_name: third_party_auth.always_associate_user_by_email
+# .. toggle_implementation: WaffleSwitch
+# .. toggle_default: False
+# .. toggle_description: Always associates current social auth user with
+#   the user with the same email address in the database, which verifies
+#   that only a single database user is associated with the email.
+# .. toggle_use_cases: opt_in
```

Comment:
@nizarmah: When I wrote [my last comment](https://github.com/edx/edx-platform/pull/25935/files#r567959945), I thought this was part of the docathon effort.  The comment still stands, but I didn't realize this was already fully reviewed, so I don't plan on reviewing.

However, according the the review comments, it seems like this toggle is `temporary`, and just being used for rollout to ultimately switch over the the newer way.  I may have misunderstood though.  If this is true, please switch to `temporary` and maybe note that this is just being used for rollout. If I misunderstood the other comments on the PR, you can correct me and ignore this. Thanks.